apps/api/src/app/topics-v1/use-cases/add-subscribers/add-subscribers.use-case.ts

```diff
@@ -98,6 +98,7 @@ export class AddSubscribersUseCase {
       _topicId: topic._id,
       topicKey: topic.key,
       externalSubscriberId: subscriber.subscriberId,
+      identifier: `tk=${topic.key}:si=${subscriber.subscriberId}`,
```

Comment:
```suggestion
      identifier: `tk_${topic.key}:si_${subscriber.subscriberId}`,
```

Just to make it easier encode uri components

Reply:
done, i created a centralized function to for the default identifier so we will have it create only with underscore.